### ISSUE TITLE
aiobotocore update to 1.4.0  removed top level AioSession name, resulting in module 'aiobotocore' has no attribute 'AioSession'

### ISSUE THREAD

**Issue by dhuettenmoser** [dask/s3fs#512]
getting an error "module 'aiobotocore' has no attribute 'AioSession'" when using S3FileSystem(anon=False) 

Release 1.4.0 of aiobotocore seems to be the cause:

from https://pypi.org/project/aiobotocore/ 
    1.4.0 (2021-08-20)
      -  fix retries via config #877
      -     remove AioSession and get_session top level names to match botocore
      -     change exceptions raised to match those of botocore, see mappings

```
s3 = S3FileSystem(anon=False)
s3.open(f'{bucket}/{key}', mode='rb')
```

**Comment by martindurant** [comment]
Fixed in #510 - I will schedule a release shortly.

**Comment by dhuettenmoser** [comment]
Thanks!
